packages/date-parser/src/parsers/lastX.js

```diff
@@ -19,7 +20,7 @@ parser.pattern = () => {
  */
 parser.extract = (text, ref, match, opt) => {
   const { weekStartDay } = opt;
-  const modifier = match[1];
+  const modifier = match[1].toLowerCase();
```

Comment:
Should we lower case the whole string before passing to Chrono?
It would reduces work for each parser

Reply:
I think this is fine.  
On the other hand, if we downcase the string:
* it will be complex to determine which parts of the original string is matched
* it will limit the flexibility of each parser